modules/microshift-firewall-update-for-service.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-firewall.adoc
+
+:_content-type: CONCEPT
+[id="microshift-firewall-update-for-service_{context}"]
+= Updating firewall ports when a service is exposed
+
+Firewalld is often active when you run services on a {product-title}. This can disrupt certain services on a {product-title} because traffic to the ports might be blocked by the firewall. You need to ensure the necessary firewall ports are open if you want certain services to be accessible from outside the host. There are several options for opening your ports:
+
+* Services of the `NodePort` and `LoadBalancer` type are automatically available with OVN-K.
++
+In these cases, OVN-K adds iptable rules so the traffic to the cluster IP address is delivered to the relevant ports. This is done using the PREROUTING rule chain and is then forwarded to the OVN-K to bypass the firewalld rules for local host ports and services.
```

Comment:
Maybe we shall add some background knowledge like: 
Both iptables and firewalld are backed by nftables in RHEL9. To avoid conflict, the nftables rules generated by iptables always have precedence over the ones generated by firewalld.